lib/pbench/test/unit/server/test_shell_cli.py

```diff
@@ -189,10 +189,16 @@ def run(args, cwd: Optional[str] = None) -> subprocess.CompletedProcess:
     def test_main(
         monkeypatch, make_logger, mock_get_server_config, user_site, oidc_conf
     ):
-        find_called = [False]
+        called = []
 
         def find_the_unicorn(logger: logging.Logger):
-            find_called[0] = True
+            called.append("find_the_unicorn")
+
+        def wait_for_uri(*args, **kwargs):
+            called.append("wait_for_uri")
```

Comment:
Consider including the URI argument in the `called` message, so that we can tell one call from the other, now that there should be two.

Reply:
Fixed.